### ISSUE TITLE
Add an option to analyze a pcap file

### ISSUE THREAD

**Issue by ghost** [alagoa/youtube-or-pornhub#3]
It would be awesome if the tool could analyze a pre-sniffed pcap file, and not only real-time detection of events.

**Comment by alagoa** [comment]
Yep, that would be great. I'll start my master thesis on this subject in about 2 months so I'll probably add that feature, it'll be useful for testing stuff. Thanks!
